backends/p4tools/modules/testgen/targets/bmv2/test_backend/common.h

```diff
@@ -0,0 +1,43 @@
+#ifndef BACKENDS_P4TOOLS_MODULES_TESTGEN_TARGETS_BMV2_TEST_BACKEND_COMMON_H_
+#define BACKENDS_P4TOOLS_MODULES_TESTGEN_TARGETS_BMV2_TEST_BACKEND_COMMON_H_
+
+#include <filesystem>
+#include <optional>
+#include <vector>
+
+#include <inja/inja.hpp>
+
+#include "backends/p4tools/modules/testgen/lib/test_object.h"
+#include "backends/p4tools/modules/testgen/lib/test_spec.h"
+#include "backends/p4tools/modules/testgen/lib/tf.h"
+
+namespace P4Tools::P4Testgen::Bmv2 {
+
+class Bmv2TF : public TF {
+ public:
+    explicit Bmv2TF(std::filesystem::path basePath,
+                    std::optional<unsigned int> seed = std::nullopt);
+
```

Comment:
I'm quite surprised to not see a virtual OutputTest here. Can you maybe say a bit why that doesn't belong?

Reply:
`outputTest` is already purely virtual in `TF`, but every test back end instantiation is slightly different. This makes it not worthwhile to provide a common, shared function.

Reply:
Ah, I see! That makes sense. Forgot that Bmv2TF inherits from TF! Thanks!